cmd/cloud/installation.go

```diff
@@ -615,3 +620,101 @@ var installationRecoveryCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var installationDeploymentReportCmd = &cobra.Command{
+	Use:   "report",
+	Short: "Get a report of deployment details for a given installation",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		installationID, _ := command.Flags().GetString("installation")
+
+		installation, err := client.GetInstallation(installationID, &model.GetInstallationRequest{
+			IncludeGroupConfig:          true,
+			IncludeGroupConfigOverrides: true,
+		})
+		if err != nil {
+			return errors.Wrap(err, "failed to query installation")
+		}
+		if installation == nil {
+			return nil
+		}
+
+		output := fmt.Sprintf("Installation: %s\n", installation.ID)
+		output += fmt.Sprintf(" ├ Created: %s\n", installation.CreationDateString())
+		output += fmt.Sprintf(" ├ State: %s\n", installation.State)
+		if installation.State == model.InstallationStateDeleted {
+			output += fmt.Sprintf(" │ └ Deleted: %s\n", installation.DeletionDateString())
+		}
+		output += fmt.Sprintf(" ├ DNS: %s\n", installation.DNS)
+		output += fmt.Sprintf(" ├ Size: %s\n", installation.Size)
+		output += fmt.Sprintf(" ├ Affinity: %s\n", installation.Affinity)
+		output += fmt.Sprintf(" ├ Environment Variables: %d\n", len(installation.MattermostEnv))
+		output += fmt.Sprintf(" ├ Database Type: %s\n", installation.Database)
+		if model.IsMultiTenantRDS(installation.Database) {
+			databases, err := client.GetMultitenantDatabases(&model.GetDatabasesRequest{
+				DatabaseType: model.DatabaseEngineTypePostgresProxy,
+				Paging: model.Paging{
+					Page:           0,
+					PerPage:        model.AllPerPage,
+					IncludeDeleted: true,
+				},
```

Comment:
Could be slightly simplified:
```suggestion
				Paging: model.AllPagesWithDeleted(),
```